ui/components/package.json

```diff
@@ -43,6 +43,7 @@
     "lodash-es": "^4.17.21",
     "mathjs": "^10.6.4",
     "mdi-material-ui": "^7.4.0",
+    "react-colorful": "^5.6.1",
```

Comment:
I was choosing between either [react-color](https://github.com/casesandberg/react-color) or  [react-colorful](https://github.com/omgovich/react-colorful), the two most popular color picker libraries.

I ended up choosing `react-colorful` because the design is simple (@foxbat07 approved), package is smaller, and it is built with react hooks and functional components. It ended up being very easy to use :)